battle-service/src/main/java/com/adavydenko/tictactoe/battleservice/entities/Coordinate.java

```diff
@@ -0,0 +1,12 @@
+package com.adavydenko.tictactoe.battleservice.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Coordinate {
+    A(0), B(1), C(2);
```

Comment:
Does ot supposed to be static grid?

I suppose yes.
If it is so, let's consider that the battle area could be configurable and grid could have square form (3x3 5x5,10x10 etc) more over, lets consider that figures in a row to win can also be configurable ( 5,4,6 etc) but by default will take x(or y) size of the square